### ISSUE TITLE
Coor

### ISSUE THREAD

**Issue by dersim-davaod** [thaliproject/Thali_CordovaPlugin#1343]
https://github.com/ThaliTester/TestResults/tree/89624796e90776b_Fix_issues_with_native_unit_tests_on_iOS__dersim-davaod/

**Comment by yaronyg** [comment]
@dersim-davaod But are they failing locally?

**Comment by dersim-davaod** [comment]
No, we can run tests locally, however, they fail on CI (see logs above).
Anyway currently it makes no sense to run them on CI, so we decided to disable them in `iOS` branch (issue #1353.)
